### DISCUSSION TITLE
Reference Architecture without Keybase

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#485] [Help] [answered]
Looking at https://github.com/gruntwork-io/knowledge-base/discussions/337, my organization doesn't allow me to have a Keybase account; however, I do have AWS SSO, which means I can insert roles to assume via permission sets to get into the Security account. Is it possible to not use Keybase since I don't need an IAM User password in this instance? 

---

<ins datetime="2022-06-24T19:56:23Z">
  <p><a href="https://support.gruntwork.io/hc/requests/108834">Tracked in ticket #108834</a></p>
</ins>

**Comment by yorinasub17** [comment, marked as answer]
You can use a PGP key in place of keybase. Once you have created a keypair using `gpg`, you can export it into a format that Terraform understands using the command provided [here](https://github.com/gruntwork-io/terraform-aws-security/tree/master/modules/iam-users#how-do-i-use-pgp-keys-directly).
